net/tor/src/opnsense/mvc/app/models/OPNsense/Tor/Menu/Menu.xml

```diff
@@ -0,0 +1,7 @@
+<menu>
+  <Services>
+    <tor VisibleName="Tor" cssClass="fa fa-bolt fa-fw">
+      <configuration VisibleName="Configuration" url="/ui/tor/" />
```

Comment:
Already space for log file, or?

Reply:
It will probably need something to fetch data like the hostnames for hidden services etc.
Maybe also a log view (even if tor should not log too much).

Reply:
ok